client/src/utils/useForm.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+
+
+const useForm = () => {
+    const [state, setState] = useState({});
```

Comment:
Maybe useForm could take a config param to set it's base state to in cases where for instance you would need an empty array assigned to a value in state.